starwizard-services/src/main/java/io/stardog/starwizard/services/media/info/ImagemagickInfo.java

```diff
@@ -0,0 +1,26 @@
+package io.stardog.starwizard.services.media.info;
+
+import io.stardog.starwizard.services.media.interfaces.ImageInfo;
+import org.im4java.core.Info;
+import org.im4java.core.InfoException;
+
+import javax.inject.Singleton;
+
+@Singleton
+public class ImagemagickInfo implements ImageInfo {
```

Comment:
I think I would name these: `ImagemagickInfoService` and `ImageInfoInterface` to reflect the fact that the class itself does not represent "info", but a service to retrieve info.